lib/semian/memcached.rb

```diff
@@ -0,0 +1,67 @@
+require 'semian/adapter'
+require 'memcached'
+
+module Semian
+  module Memcached
+    include Semian::Adapter
+
+    class SemianError < ::Memcached::Error
+      def initialize(semian_identifier, *args)
+        super(*args)
+        @semian_identifier = semian_identifier
+      end
+    end
+
+    ResourceBusyError = Class.new(SemianError)
+
+    def semian_identifier
+      name = semian_options && semian_options[:name]
+      ["memcached", name].compact.join("_").to_sym
+    end
+
+    def raw_semian_options
+      # disable semian when host ejection is enabled.
+      return false if !!@options[:auto_eject_hosts]
```

Comment:
Why does this matter? Since the ticket is taken for the group of servers and not for a specific server with the current implementation.

Reply:
Also, I'm not sure why you are using `!!` in an `if` condition.

Reply:
Yep that's my bad, forgot that from my previous experiments.